data/data/bootstrap/bootstrap-in-place/files/usr/local/bin/bootstrap-in-place.sh

```diff
@@ -16,7 +16,7 @@ if [ ! -f stop-etcd.done ]; then
   echo "Stop etcd static pod by moving the manifest"
   mv /etc/kubernetes/manifests/etcd-member-pod.yaml /etc/kubernetes || echo "already moved etcd-member-pod.yaml"
 
-  until ! crictl ps | grep etcd; do
+  until ! crictl ps --name etcd | grep etcd; do
```

Comment:
`--name etcd` might still capture `etcd` in container names which might contain the hostname, maybe we should work with structured output instead:

```suggestion
  until sudo crictl ps --state running -o json | jq -e '[.containers[] | select(.metadata.name == "etcd")] | length == 0'; do
```

Reply:
I'm sure that the container name during bootstrap is just etcd, yet this jq seems more robust

Reply:
The etcd container name is just etcd, but other containers might be running and might have the hostname in their name